lib/item.go

```diff
@@ -92,8 +91,9 @@ func (a Items) Less(i, j int) bool { return a[i].ID < a[j].ID }
 
 func (a Items) At(i int) IDCarrier { return a[i] }
 
-func (item Item) DateTime() time.Time {
+func (item Item) DateTime() (time.Time, bool) {
 	var date string
+	var all_day bool
```

Comment:
Can you fix this to lowerCamelCase ?
Yes, there is many wrong case variables.
But I want to unify to lowerCamelCase.